test/addons/csi-addons/start

```diff
@@ -12,7 +12,7 @@ from drenv import cache
 
 def deploy(cluster):
     print("Deploying csi addon for volume replication")
-    path = cache.get(".", "addons/csi-addons-0.12.0.yaml")
+    path = cache.get(".", "addons/csi-addons-latest.yaml")
```

Comment:
You changed the kustomization but the cache name is the same. This make the CI run invalid, using the previous version. Use "addons/csi-addons-latest-1.yaml" so the CI test the current code.